website/config.toml

```diff
@@ -21,6 +21,25 @@ disableShortcutsTitle = true
 home = [ "HTML", "AMP", "RSS", "JSON"]
 page = [ "HTML", "AMP"]
 
+[markup]
```

Comment:
Is there a way to add a comment saying that this is only needed for Hugo versions >= 0.60 (or whatever the exact number of Hugo version is)

Reply:
I can take this out [markup] , this is my local change

Reply:
ok.